src/org/linphone/InCallActivity.java

```diff
@@ -1880,7 +1889,7 @@ private void hangUp() {
 			lc.terminateAllCalls();
 		}
 		delete_messages();
-
+		g.app_killed=false;
```

Comment:
it seems hangup() is not called when remote user ends the call, so g.app_killed may not keep state correctly and the next call will use mute settings for previews call.